lib/src/widgets/device_icon.dart

```diff
@@ -7,6 +7,8 @@ const yaruIcons = <String, IconData>{
   'drive-harddisk': YaruIcons.drive_harddisk,
   'input-gaming': YaruIcons.input_gaming,
   'preferences-desktop-keyboard': YaruIcons.input_keyboard,
+  'touchpad-disabled': YaruIcons.fingerprint,
+  'thunderbolt': YaruIcons.thunderbolt
```

Comment:
nice :+1: there's a lot more if you have time and interest :smile: 

https://lvfs.readthedocs.io/en/latest/metainfo.html#device-icons

Reply:
woah, thanks!